packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -473,37 +480,54 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
             {children}
           </svg>
         </FocusZone>
+
         {!this.props.hideLegend && (
-          <div ref={(e: HTMLDivElement) => (this.legendContainer = e)} className={this._classNames.legendContainer}>
-            {this.props.legendBars}
-          </div>
+          <React.Suspense fallback={<div>Loading...</div>}>
+            <div ref={(e: HTMLDivElement) => (this.legendContainer = e)} className={this._classNames.legendContainer}>
+              {this.props.legendBars}
+            </div>
+          </React.Suspense>
+        )}
+        {!this.props.hideTooltip && calloutProps!.isCalloutVisible && (
```

Comment:
The deleted comment says that the callout is used for narration. Do you know what it means

Reply:
I think it means that previously we might have not added the logic of opening the callout ( using the tabs and arrow keys) , so it might have added the callout to dom with hidden as true so that it is accessible.

Reply:
Lets add the comment back.